common/app/views/fragments/amp/outbrain.scala.html

```diff
@@ -4,13 +4,17 @@
 
 @if(OutbrainOnAmp.isSwitchedOn) {
     @defining(page.metadata.webUrl) { htmlUrl =>
-        <amp-iframe width=100 height=100
-        sandbox="allow-top-navigation allow-scripts allow-same-origin allow-popups allow-popups-to-escape-sandbox"
-        layout="responsive"
+        <amp-iframe height=210
+        sandbox="allow-scripts allow-same-origin allow-popups"
+        layout="fixed-height"
         frameborder="0"
         resizable
-        src="https://widgets.outbrain.com/hub/amp.html#widgetIds=AMP_1&htmlURL=@URLEncode(htmlUrl)&ampURL=@URLEncode(s"$htmlUrl/amp")&styleFile=">
-            <div overflow tabindex=0 role=button aria-label="Outbrain Recommendations">Outbrain</div>
+        src="https://widgets.outbrain.com/hub/amp.html#widgetIds=AMP_1&htmlURL=@URLEncode(htmlUrl)&ampURL=@URLEncode(s"$htmlUrl?amp")"
```

Comment:
Minor detail: ampersands are special characters in HTML, you should use the entity reference: `&amp;`